index.html

```diff
@@ -12,12 +12,13 @@
       <a href="index.html"><span class="my-name">Biruk Bereka</span></a>
       <nav>
         <div class="menu"><img src="images/Enabled.png" alt="menu icon"/></div>
+        <div class="close-button invisible"><img src="./images/cancel-icon.png" alt="close menu"/></div>
         <ul class="nav-items">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#contact">Contact</a></li>
+          <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-item"><a href="#about">About</a></li>
+          <li class="nav-item"><a href="#contact">Contact</a></li>
           <li>
```

Comment:
Good job on implementing the mobile menu properly 👌It looks fantastic. But you can improve it by following the exact Figma design spacing values. Currently, your design has some extra space please make sure to reduce it.

Yours:
![image](https://user-images.githubusercontent.com/34779928/208513888-91e9dbb5-9e7a-431c-a23c-b6d4bdbfb28c.png)

Figma:
![image](https://user-images.githubusercontent.com/34779928/208513957-165055f5-84c1-4d34-aa8f-27293c93179a.png)

Note: in the Figma template the top section has a mobile status bar, you can skip its size on your design. And you can also reduce the space from the left and right sides.